JPS_Ontology/ontology/ontoems/OntoEnvironmentalMeasurementStation.csv

```diff
@@ -37,6 +39,7 @@ Visibility,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Dista
 UVIndex,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Number,IS-A,,,,The strength of the sun's ultraviolet (UV) radiation expressed as index as developed by the World Health Organization.,https://www.theworldavatar.com/kg/ontoems/,UV index
 PrecipitationProbability,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Number,IS-A,,,,The likelihood of occurrence of precipitation.,https://www.theworldavatar.com/kg/ontoems/,Precipitation probability
 PrecipitationProbability,Class,http://semanticscience.org/resource/SIO_000638,IS-A,,,,The likelihood of occurrence of precipitation.,https://www.theworldavatar.com/kg/ontoems/,Precipitation probability
+PrecipitationRate,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Speed,IS-A,,,,The amount of rain that has fallen per unit time (usually given in units of mm/hr or in/hr (millimeters per hour or inches per hour)) .,https://www.theworldavatar.com/kg/ontoems/,Precipitation rate
```

Comment:
Is there any better match for this? Speed seems correct wrt the given units; however, isn't it actually a volume per time unit and only "reduced" to length per time unit for standardised tippping bucket etc. measurement devices?

Reply:
Hmmm the reason for adding precipitation rate as a subclass of om:Speed is because I need to represent this weather station that reports precipitation rate values with mm/hr as the unit. And since mm/hr is a speed unit, it is only applicable to om:Speed and it's subclasses.

Reply:
Or will it be better to rename the concept as rainfall rate instead of precipitation rate?

Reply:
If there is nothing more specific available in om, I believe `PrecipitationRate` and Speed is fine